scheds/rust/scx_rusty/src/main.rs

```diff
@@ -623,12 +583,11 @@ impl<'a> Scheduler<'a> {
         );
 
         lb.load_balance()?;
+        self.metrics.processing_duration.record(started_at.elapsed().as_micros() as f64);
```

Comment:
I moved capturing this metric at close as possible to the function we are instrumenting. This way we don't need to pass the duration around. Likewise for `cpu_busy` a few lines above.